@navikt/core/react/src/util/i18n/locales/en.ts

```diff
@@ -37,4 +37,7 @@ export default {
       labelSuffix: "delete",
     },
   },
+  Modal: {
```

Comment:
Further on the discussion around "generic" translations [mentioned here](https://github.com/navikt/aksel/pull/3281#discussion_r1818682329). Should more generic words like: Expand, Collapse, Open, Close, Next, Previous ++ be under a generic key?

Reply:
Ignore this